core/store/ledgerstore/tx_handler.go

```diff
@@ -121,6 +124,14 @@ func HandleDeployTransaction(store store.LedgerStore, overlay *overlaydb.Overlay
 	}
 
 	notify.Notify = append(notify.Notify, notifies...)
+	notify.Notify = append(notify.Notify,
+		&event.NotifyEventInfo{
+			ContractAddress: address,
+			States: &message.ContractEvent{
```

Comment:
`ContractEvent` is too general, rename  to `DeployContractEvent`.

Reply:
ok